airflow/cli/commands/variable_command.py

```diff
@@ -23,18 +23,29 @@
 from json import JSONDecodeError
 
 from airflow.cli.simple_table import AirflowConsole
+from airflow.configuration import get_custom_secret_backend
 from airflow.models import Variable
 from airflow.utils import cli as cli_utils
 from airflow.utils.cli import suppress_logs_and_warning
 from airflow.utils.session import create_session
+from airflow.utils.warnings import warn_list_secrets_alternative_backend
+
+CUSTOM_BACKEND: bool = True if get_custom_secret_backend() else False
```

Comment:
This shouldn’t need to be repeated. Should we instead extract this to somewhere shared between both modules?